### PR TITLE
Add compats to Project.toml.

### PR DESCRIPTION
**Pull request by pkofod**

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -18,6 +18,15 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
 julia = "1"
+Calculus = "0.3.1"
```
Comment (fredrikekre) [inline Project.toml]: Does it not work with the current Calculus version? Maybe you want
```suggestion
Calculus = "0.3.1, 0.4"
```

File: Project.toml [hunk 2]
```diff
@@ -18,6 +18,15 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
...
+DiffEqDiffTools = "0.12"
```
Comment (fredrikekre) [inline Project.toml]:
```suggestion
DiffEqDiffTools = "0.12, 0.13"
```

File: Project.toml (hunk 1 above)
Comment (pkofod) [reply Project.toml]: I wrote 3 by mistake, meant 4 :)

**Comment by pkofod** [comment]
Thanks @fredrikekre ! The DiffEqDiffTools was me forgetting the "left most non-zero", that is, it isn't just the actual major version that determines the upper bound, and the Calculus one was a typo.

**Pull request merged**